ios/sdk/src/picture-in-picture/PiPViewCoordinator.swift

```diff
@@ -136,14 +152,20 @@ public class PiPViewCoordinator {
         let exitSelector = #selector(toggleExitPiP)
         tapGestureRecognizer?.removeTarget(self, action: exitSelector)
         tapGestureRecognizer = nil
-        
+
         delegate?.exitPictureInPicture()
     }
 
     /// Reset view to provide bounds, use this method on rotation or
     /// screen size changes
     public func resetBounds(bounds: CGRect) {
         currentBounds = bounds
+
+        // Is required because otherwise the pip window is buggy when rotating the device.
+        // When not in pip then autoresize will do the job.
+        if (isInPiP) {
+            view.frame = changeViewRect()
```

Comment:
I tried out removing the `autoresizingMask` and setting this here in every case, however then the underlying view instead of a black background were visible at the edges during device rotation.